runtime/libpgmath/lib/common/pow/fma3/sdpow.cpp

```diff
@@ -18,6 +18,8 @@
 
 #if defined(TARGET_LINUX_POWER)
```

Comment:
runtime/libpgmath/lib/common/pow/fma3/sdpow.cpp:19:0: error: unterminated #if
 #if defined(TARGET_LINUX_POWER)

Did this file (and it looks like a few more) build for you?

Reply:
Apparently not, but I didn't get any errors either.

I will fix this right now.